### PR TITLE
Add/fix some chunk mappings

### PR DESCRIPTION
**Pull request by mezz**

### CODE REVIEW

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 1]
```diff
@@ -57,18 +67,24 @@ CLASS none/atp net/minecraft/world/chunk/Chunk
...
 		ARG 0 pos
+		ARG 1 mode
 	METHOD a (Lnone/es;IZ)V
 		ARG 0 buffer
 	METHOD a addEntity (Lnone/sg;)V
 		ARG 0 entity
 	METHOD a removeEntity (Lnone/sg;I)V
 		ARG 0 entity
 		ARG 1 chunkY
+	METHOD a getEntitiesAndPartsInAreaExcludingOne (Lnone/sg;Lnone/bcs;Ljava/util/List;Lcom/google/common/base/Predicate;)V
```
Comment (Darkhax) [inline mappings/net/minecraft/world/chunk/Chunk.mapping]: This seems like a very long name. Would something like `getEntitiesInAreaExcluding` be an accurate name for it?

Comment (mezz) [reply mappings/net/minecraft/world/chunk/Chunk.mapping]: It's an unusually bad function so I feel like it requires the long name.
This gives people an idea of what it's used for without having to puzzle it out by reading the code.

Comment (asiekierka) [reply mappings/net/minecraft/world/chunk/Chunk.mapping]: I don't think we need a name this long. Parameters can also have clues.

Comment (mezz) [reply mappings/net/minecraft/world/chunk/Chunk.mapping]: changed to `getEntitiesInArea`

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 2]
```diff
@@ -57,18 +67,24 @@ CLASS none/atp net/minecraft/world/chunk/Chunk
...
+		ARG 1 mode
 	METHOD a (Lnone/es;IZ)V
 		ARG 0 buffer
 	METHOD a addEntity (Lnone/sg;)V
 		ARG 0 entity
 	METHOD a removeEntity (Lnone/sg;I)V
 		ARG 0 entity
 		ARG 1 chunkY
+	METHOD a getEntitiesInArea (Lnone/sg;Lnone/bcs;Ljava/util/List;Lcom/google/common/base/Predicate;)V
+		ARG 0 excludedEntity
```
Comment (copygirl) [inline mappings/net/minecraft/world/chunk/Chunk.mapping:84]: There's a `sendToAllPlayers` that sends packets as well where I named the "excluded player" parameter `except`. For consistency's sake, should I rename this one to `except` or the packet one to `excludedPlayer` ?

**Pull request merged**